types/cordova-plugin-inappbrowser/cordova-plugin-inappbrowser-tests.ts

```diff
@@ -5,7 +5,15 @@
 // is similar to native window.open signature, so the compiler can's
 // select proper overload, but we cast result to InAppBrowser manually.
 var iab = <InAppBrowser>window.open('google.com', '_self');
-iab.addEventListener('loadstart', (ev: InAppBrowserEvent) => { console.log('Start opening ' + ev.url); });
+iab.addEventListener('loadstart', (ev) => { console.log('loadstart' + ev.url); });
```

Comment:
gotcha. I will have it both. I remove it to make sure that contextual type work (so we only get test pass if `ev` is `InAppBrowserEvent`